### PR TITLE
Validation form implemented

### PR DESCRIPTION
**Pull request by diegoh40**
In this part of the project we implemented a concept about validation in the form, so we could apply the JS function and DOM knowledge for this activity.

### CODE REVIEW

File: js/scripts.js [hunk 1]
```diff
@@ -142,6 +137,26 @@ exampleModal.addEventListener('show.bs.modal', (event) => {
...
+
+  const emailVal = email.value.trim();
+
+  if (!emailRegex.test(emailVal)) {
+    document.getElementById('alertWin').style.display = 'block';
+
+    email.focus();
+  } else {
+    form.submit();
+  }
```
Comment (sja-thedude) [inline js/scripts.js:157]: - [ ] Kindly make sure that when a user submits the contact form with the email field containing uppercase characters an error message should show up. Like `Only lowercase allowed for email` or `Use only lowercase in email field`.
- [ ] Kindly make sure that the error message should show up close to the "submit" button.
To understand this better you can view this loom video given [here](https://www.loom.com/share/2b59121265f54e10903f8d402b088870).

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @diegoh40, This is @sja-thedude your code reviewer.

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️Correct git-flow
✔️Professional readme
✔️Good job so far

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by diegoh40** [comment]
fixed the alert message that my code reviewer kindly suggested that I could improve in my code, now it does not show a top-bottom alert but also an empty field alert, this way everything is working better.

**Review by zieeco — APPROVED** [review]
# status: APPROVED :green_circle: :100: 

Hi, @diegoh40 


68747470733a2f2f6d656469612e67697068792e636f6d2f6d656469612f78543078657a514755357843444a754350652f67697068792e676966


Good job so far 👍 Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


## Highlights ⚡ 
✅ Descriptive PR 👍 
✅ Nice design 🔥 
✅ Linters are passing 👏 


## Optional suggestions
- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**